README.md

```diff
@@ -224,14 +224,19 @@ keys:
   - &admin_alice 2504791468b153b8a3963cc97ba53d1919c5dfd4
   - &admin_bob age12zlz6lvcdk6eqaewfylg35w0syh58sm7gh53q5vvn7hd7c6nngyseftjxl
 creation_rules:
-  - path_regex: secrets/[^/]+\.yaml$
+  - path_regex: secrets/[^/]+\.(yaml|json|env|ini)$
     key_groups:
     - pgp:
       - *admin_alice
       age:
       - *admin_bob
 ```
 
+**Note:** be sure to not include a `-` before subsequent key types under `key_groups`
+(i.e. `age` in the above example should not have a `-` in front). If you do, `sops`
+will produce [a vague error](https://github.com/getsops/sops/issues/857) when
+encrypting files.
+
```

Comment:
I think this is a common pitfall even I went into a couple of times.
I would extends this a bit to capture more what is actually the difference between the two is. What do you think about the following?

> **Note** be sure to not include a `-` before subsequent key types under `key_groups` (i.e. `age` in the above example should not have a `-` in front).
This will otherwise cause sops to require multiple keys (shamir secret sharing) to decrypt a secret, which breaks normal sops-nix usage.

Reply:
Oh TIL that syntax becomes rendered in GitHub-flavoured markdown! And that it was not an invalid configuration I did, but just one that I didn't intend.

I like your suggestion, and have applied it in e4e1a9f.